aws-ssm-patchbaseline/src/main/java/software/amazon/ssm/patchbaseline/UpdateHandler.java

```diff
@@ -112,15 +117,33 @@ public UpdateHandler(TagHelper tagHelper) {
             logger.log(String.format("INFO Updated tags for patch baseline %s %n", baselineId));
 
             // Set to default patch baseline
-            if (BooleanUtils.isTrue(model.getDefaultBaseline()) && !BooleanUtils.isTrue(previousModel.getDefaultBaseline())){
+           // if (BooleanUtils.isTrue(model.getDefaultBaseline()) && !BooleanUtils.isTrue(previousModel.getDefaultBaseline())){
+            if(model.getDefaultBaseline()!=null && BooleanUtils.isTrue(model.getDefaultBaseline()) ){
+
                 RegisterDefaultPatchBaselineRequest registerDefaultPatchBaselineRequest = RegisterDefaultPatchBaselineRequest.builder()
                         .baselineId(baselineId)
                         .build();
                 RegisterDefaultPatchBaselineResponse registerDefaultPatchBaselineResponse =
                         proxy.injectCredentialsAndInvokeV2(registerDefaultPatchBaselineRequest, ssmClient::registerDefaultPatchBaseline);
 
-                logger.log(String.format("INFO Registered patch baseline %s to default patch baseline successfully %n", baselineId));
+                if(registerDefaultPatchBaselineResponse== null){
+
+                    return ProgressEvent.<ResourceModel, CallbackContext>builder()
```

Comment:
why the return here?

Reply:
Here i have return the empty model data to the handler since no update changes are not happend in default patch baseline mode.
Some time we will create an patch baseline without default patch baseline updating requirement in that case we will return with model with empty default patch baseline details.  
This senarios's was validated in unit test & contract test case it working as good